packages/react-instantsearch-core/src/connectors/connectConfigure.js

```diff
@@ -16,12 +16,12 @@ export default createConnector({
     return {};
   },
   getSearchParameters(searchParameters, props) {
-    const items = omit(props, 'children', 'contextValue');
+    const items = omit(props, 'children', 'contextValue', 'indexContextValue');
     return searchParameters.setQueryParameters(items);
   },
   transitionState(props, prevSearchState, nextSearchState) {
     const id = getId();
-    const items = omit(props, 'children', 'contextValue');
+    const items = omit(props, 'children', 'contextValue', 'indexContextValue');
```

Comment:
We have to add a test for this behaviour.

Reply:
added a test in 8fa0d684